components/claim/components/email_to_oauth.tsx

```diff
@@ -40,24 +53,24 @@ export default class EmailToOAuth extends React.PureComponent {
 
         this.setState({password});
 
-        state.error = null;
+        // state.error = null;
```

Comment:
there is no need to leave any commented code

Reply:
OK I will delete this also across PR